dataflow/src/main/java/org/checkerframework/dataflow/expression/JavaExpressionConverter.java

```diff
@@ -34,8 +34,13 @@ public JavaExpression convert(JavaExpression javaExpr) {
      * @return the list of converted expressions
      */
     public List<@PolyNull JavaExpression> convert(List<@PolyNull JavaExpression> list) {
-        return CollectionsPlume.mapList(
-                (JavaExpression expression) -> expression == null ? null : convert(expression),
+        return SystemUtil.mapList(
+                (@PolyNull JavaExpression expression) -> {
+                    if (expression == null) {
```

Comment:
I'm curious why a ternary operator does not work here.

Reply:
https://github.com/typetools/checker-framework/issues/1170.

Reply:
Thanks.  I would add a reference to it.  (That will also allow us to clean this up when the issue is fixed.)